LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2596,6 +2596,43 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chipset Fan", 3));
                         break;
 
+                    case Model.Z690_Extreme:
+                        v.Add(new Voltage("Vcore", 0, 10, 10));
+                        v.Add(new Voltage("+5V", 1, 20, 10));
+                        v.Add(new Voltage("AVCC", 2, 34, 34));
+                        v.Add(new Voltage("+3.3V", 3, 34, 34));
+                        v.Add(new Voltage("+12V", 4, 110, 10));
+                        v.Add(new Voltage("VCCIN AUX", 5, 1, 1));
+                        v.Add(new Voltage("DRAM", 6));
+                        v.Add(new Voltage("+3.3V Standby", 7, 34, 34));
+                        v.Add(new Voltage("VBat", 8, 34, 34));
+                        v.Add(new Voltage("VTT", 9));
+                        v.Add(new Voltage("1.05V CPU Voltage", 10, 1, 1));
+                        v.Add(new Voltage("0.82V Chipset Voltage", 11, 1, 1));
+                        v.Add(new Voltage("1.0V Chipset Voltage", 12));
+                        v.Add(new Voltage("CPU SA", 13, 1, 1));
+                        v.Add(new Voltage("VIN7", 14)); // Still trying to identify what this is
+                        v.Add(new Voltage("VIN9", 15)); // Still trying to identify what this is
```

Comment:
```suggestion
                        v.Add(new Voltage("Voltage #14", 14));
                        v.Add(new Voltage("Voltage #15", 15));
```

Reply:
I just updated this to use the generic voltage names as you suggested, and made these hidden by default.